src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  // Логика скидок
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT; // Скидка $50 для 7 или более дней
```

Comment:
The discount logic here applies a flat $50 discount for rentals of 7 or more days. If the task requires a different discount calculation (e.g., a percentage or a per-day discount), this logic will need to be adjusted.